airflow/utils/log/logging_mixin.py

```diff
@@ -58,21 +61,23 @@ def _set_context(self, context):
             set_context(self.log, context)
 
 
-# TODO: Formally inherit from io.IOBase
```

Comment:
Why did you remove this todo? It should still be done in the future, no?

Reply:
My mistake, will revert